contrib/opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -1012,6 +1017,17 @@ def test_aaf_composition_metadata_mastermob(self):
 
         self._verify_user_comments(aaf_metadata, expected_md)
 
+    def test_aaf_multiple_timecode_objects(self):
+        """
+        Make sure we can read AAFs with multiple timecode objects of the
+        same start value and length.
+        """
+
+        timeline = otio.adapters.read_from_file(
+            MULTIPLE_TIMECODE_OBJECTS_PATH)
+
+        self.assertTrue(timeline is not None)
```

Comment:
There is a nice unit test assertion for this that is more direct:
```suggestion
        self.assertIsNotNone(timeline)
```